kevlar/varmap.py

```diff
@@ -194,165 +216,40 @@ def snv_variant(self, qseq, tseq, mismatches, offset, ksize):
             globalcoord = self.cutout.local_to_global(localcoord)
             nikmers = n_ikmers_present(self.contig.ikmers, altwindow)
             snv = VariantSNV(
-                self.seqid, globalcoord, refr, alt, VW=altwindow,
-                RW=refrwindow, IK=str(nikmers)
+                self.seqid, globalcoord, refr, alt, CONTIG=qseq,
+                CIGAR=self.cigar, KSW2=str(self.score), IKMERS=str(nikmers),
+                ALTWINDOW=altwindow, REFRWINDOW=refrwindow
             )
             yield snv
 
-    def call_snv(self, ksize, mindist=5, logstream=sys.stderr):
-        """Call SNVs from the given alignment."""
-        length = int(self.alnmatch.group(4))
-        offset = self.offset
-        if self.targetshort:
-            gdnaoffset = 0
-            t = self.refrseq[:length]
-            q = self.varseq[offset:offset+length]
-        else:
-            gdnaoffset = offset
-            t = self.refrseq[offset:offset+length]
-            q = self.varseq[:length]
-        diffs = [i for i in range(length) if t[i] != q[i]]
-        if mindist:
-            diffs = trim_terminal_snvs(diffs, length, mindist, logstream)
-        if len(diffs) == 0:
-            nocall = Variant(
-                self.seqid, self.cutout.local_to_global(gdnaoffset), '.', '.',
-                QN=self.contig.name, QS=q
-            )
-            nocall.filter(vf.PerfectMatch)
-            yield nocall
-            return
-
-        for call in self.snv_variant(q, t, diffs, offset, ksize):
-            yield call
-
-    def deletion_allele(self, target, query, ksize):
```

Comment:
It is such a relief to jettison such large blocks of code! Especially when the code replacing it is so much simpler and more concise.